astro/src/content/docs/apis/passwordless.mdx

```diff
@@ -52,8 +52,10 @@ If you plan to utilize the FusionAuth login page then you will not need to use t
   <APIField name="applicationId" type="String" required>
     The unique Id of the Application you are requesting to log into.
   </APIField>
-  <LoginIdField login_id_type_field="loginIdType" />
-  <LoginIdTypeField />
+  <LoginIdField login_id_type_field="loginIdTypes" footer={
```

Comment:
Trying to explain this here, maybe it's unnecessary but seems like it would be helpful